app/models/article.rb

```diff
@@ -394,7 +394,8 @@ def index_id
   end
 
   def update_score
-    update_columns(score: reactions.sum(:points),
+    new_score = reactions.sum(:points) + Reaction.where(reactable_id: user_id, reactable_type: "User").sum(:points)
```

Comment:
Considering we are doing a reactable id and type lookup here and [below in the worker](https://github.com/thepracticaldev/dev.to/pull/6006/files#diff-cf8f683318c39fed7b700e71167ce848R13) we should probably put a composite index on the two fields.